shared/common-adapters/usernames.js

```diff
@@ -25,6 +25,23 @@ function usernameText({
   inlineGrammar = false,
   showAnd = false,
 }: Props) {
+  const andStyle = collapseStyles([
```

Comment:
we should move the static styles into a thing below and use stylesheetcreate

Reply:
I can't figure out a way to get Flow to be happy with using `styleSheetCreate`:
```
Cannot call styleSheetCreate with object literal bound to map because:
 • property textDecoration is missing in _StylesCrossPlatform [1] but exists in object literal [2] in property
   andStyle.
```

Reply:
you need to use platformStyles and not isMobile checks